### ISSUE TITLE
Halts on 'Final sync' when zcashd already running

### ISSUE THREAD

**Issue by BostonZcash** [ZbayApp/zbay#93]
**Version:** 1.0.66-dev-linux-18-2 (appimage)
**Platform:** Ubuntu 20.04 x64
**zcashd**: v3.0.0 (via Zecwallet Fullnode)

Upon first run, choosing 'use existing installation' and then clicking app's 'view progress' button shows `Final sync (0/999999) ~40 minutes left` along with a ~90% complete progress bar, though neither of which progress after an hour.

Only after shutting down my existing instance of zcashd and relaunching Zbay did startup progress to `Rescanning, ~25 minutes left`, finding that Zbay wants to run the daemon itself.

I'd suggest adding a notice along the lines of, 'We detected zcashd is already running. Please stop your node daemon so that Zbay may launch it.' or preferably a means to coexist with an already running node, as users may wish run Zecwallet Fullnode and Zbay concurrently.

**Comment by holmesworcester** [comment]
Duplicate of #112 (another internal ticket I just moved over)
